recipes/breakfest/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omellete
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
```

Comment:
I would maybe say the difficulty is _slightly_ higher, like it is super easy but requires a teeny bit of attention - I would say 10% is literally like a bowl of cereal

Reply:
Happy to go with a consensus on this tho

Reply:
yeah good point